internal/venice-common/src/main/java/com/linkedin/venice/pubsub/api/PubsubProduceResult.java

```diff
@@ -0,0 +1,33 @@
+package com.linkedin.venice.pubsub.api;
+
+/**
+ * An interface implemented by specific PubsubProducerAdapters to return the result of a produce action.
+ */
+public interface PubsubProduceResult {
```

Comment:
Let's use camel case in the `PubSub`. Can we also add the `get` prefix to all getter function names?